docs/frequently-asked-questions.rst

```diff
@@ -306,48 +306,11 @@ independent copy of the state variable is created in memory and
 is another issue). The modifications to this independent copy do not
 carry back to ``data1`` or ``data2``.
 
-A common mistake is to declare a local variable and assume that it will
```

Comment:
Do you agree that this whole section can be removed?

Reply:
Yes.